resources/service-monitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: prometheus-operator
+    release: prometheus-kubeaddons
+  name: cassandra-cluster-monitor
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+  namespaceSelector:
+    matchNames:
+    - default
```

Comment:
Should we add a param like `{{ .Params.Prometheus.Namespace }}` ? This currently assumes prom is installed under `default` (like it is with Konvoy) but that might not be the case with users on different k8s distros

Reply:
Dang, that's a good point Sam. It appears that even Kafka is [setting it statically to `"default"`](https://github.com/kudobuilder/operators/blob/53a70a657b23814a62a04495ffe72fe7869d7666/repository/kafka/docs/latest/resources/service-monitor.yaml#L14).

Let's add `SERVICE_MONITOR_PROMETHEUS_NAMESPACE` in `params.yaml`.

Reply:
As we are using kubectl create, it wont take param like that.
Also "default" is namespace of Cassandra cluster not Prometheus.

Reply:
If the `default` here is related to the Cassandra operator namespace, it should be `{{ .Namespace }}` then. I was under the impression that it was the namespace for the Prometheus instance.

And we can remove `SERVICE_MONITOR_PROMETHEUS_NAMESPACE` too.

Reply:
yea my bad forgot to remove SERVICE_MONITOR_PROMETHEUS_NAMESPACE.

Reply:
also 
https://github.com/mesosphere/kudo-cassandra-operator/pull/4#discussion_r338155501

Reply:
service-monitor.yaml is automated now as we are focusing on Prometheus only.